test/suites/operations.sh

```diff
@@ -5,7 +5,7 @@ wait_no_operations() {
   while [ "${retries}" -gt 0 ]; do
     echo "Waiting operations to complete (${retries} retries left) ..."
     count=$(lxc query "/1.0/operations?all-projects=true" | jq '.success | length')
```

Comment:
On a remote with many operations ongoing I get:
```
$ lxc query "foo:/1.0/operations?all-projects=true"
{
	"running": [
		"/1.0/operations/93d0acf7-9ab6-4e3e-919d-ff208411b206",
...
		"/1.0/operations/3d7caed8-8fa9-46ec-adfd-ea8d3dcb40b9"
	]
}

# counting **running** operations
$ lxc query "foo:/1.0/operations?all-projects=true" | jq '.running | length'
381
```

Is there a case where operations show as `success` rather than `running`?

Reply:
Operations are returned as a `map[string][]string` where the key is the operation state. When an operation completes it's state will be `success` and this will hang around for a few seconds before it is deleted from the database.

Reply:
I that case, the function's name (`wait_no_operations()`) and description feel wrong.